flink-core/src/main/java/org/apache/flink/api/common/ExecutionConfig.java

```diff
@@ -741,30 +725,6 @@ public CodeAnalysisMode getCodeAnalysisMode() {
 		return codeAnalysisMode;
 	}
 
-	/**
-	 * @deprecated Ineffective. Will be removed at 2.0.
-	 */
-	@Deprecated
-	public ExecutionConfig enableSysoutLogging() {
```

Comment:
Update the docs as well see (both en and zh): https://ci.apache.org/projects/flink/flink-docs-master/dev/execution_configuration.html

Reply:
The Pyhton API `execution_config.py` also needs an update.

Reply:
Sorry, I missed these parts :(

Reply:
Sorry, I missed these parts :(